### ISSUE TITLE
strange behavior with RTX 3080 TI under Windows 10

### ISSUE THREAD

**Issue by comet-bg** [NebuTech/NBMiner#731]
I have a test system with, 2X Gigabyte gamming oc RTX 3080 Ti 12 GB. One card allows +1200 OC memory, while the other only +600.
During the tests I noticed that I get a different hashrate when I use both devices in one miner and when I start separate miner processes for each video card.

INFO - ===================== [nbminer v40.1] Summary 2022-01-01 12:18:01 =====================
INFO - |ID|Device|Hashrate| LHR|Accept|Reject|Inv|Powr|CTmp|MTmp|Fan|CClk|GMClk|MUtl|Eff/Watt|
INFO - | 0|3080ti| 84.70 M|75.4|     3|     0|  0| 276|  55|  84| 80|1350|10702| 100| 306.9 K|
INFO - | 1|3080ti| 85.61 M|74.8|    10|     0|  0| 276|  59|  86| 80|1350|10102| 100| 310.2 K|
INFO - |------------------+----+------+------+---+----+--------------------------------------|
INFO - |    Total: 170.3 M|    |    13|     0|  0| 552| Uptime:  0D 00:10:30        CPU:  0% |

INFO - ===================== [nbminer v40.1] Summary 2022-01-01 16:16:01 =====================
INFO - |ID|Device|Hashrate| LHR|Accept|Reject|Inv|Powr|CTmp|MTmp|Fan|CClk|GMClk|MUtl|Eff/Watt|
INFO - | 0|3080ti| 88.73 M|74.5|    86|     0|  0| 263|  54|  82| 80|1350|10702| 100| 336.9 K|
INFO - |------------------+----+------+------+---+----+--------------------------------------|
INFO - |    Total: 88.73 M|    |    86|     0|  0| 263| Uptime:  0D 02:12:00        CPU:  0% |

INFO - ===================== [nbminer v40.1] Summary 2022-01-01 16:20:50 =====================
INFO - |ID|Device|Hashrate| LHR|Accept|Reject|Inv|Powr|CTmp|MTmp|Fan|CClk|GMClk|MUtl|Eff/Watt|
INFO - | 1|3080ti| 85.71 M|74.4|    12|     0|  0| 257|  58|  84| 80|1350|10102| 100| 333.5 K|
INFO - |------------------+----+------+------+---+----+--------------------------------------|
INFO - |    Total: 85.71 M|    |    12|     0|  0| 257| Uptime:  0D 00:26:00        CPU:  0% |

At the same OC settings in the first case I get a stable hashrate of about 170MH/s, while when starting in separate processes my hashrate is 175MH/s. 
In addition, I noticed that when I use an external program for OC settings (MSI AB) hashrate is stable while when using the built-in settings it varies widely, although I do not see a change in frequencies.

**Comment by AjlaaSS** [comment]
what is your memory temperatures on cards? I'm little confused from looking at list.

**Comment by comet-bg** [comment]
The temperature can be seen in the examples:
Ctmp = 55-58
Mtmp = 80-84
Rarely the memory reaches 86 degrees but the average is 84, which I think is a good value.
Another setting I made with nvidia profile inspector to fix P0 state.

**Comment by AjlaaSS** [comment]
yes that is ok temperature. so you can exclude it as reason for lower hashrate.
I suspected it maybe, if two cards are close and heating each other and throttling, but your temperatures are fine.

**Comment by comet-bg** [comment]
Continuing the tests I noticed something strange again - after applying the OC memory +1200 for ID0 and +600 for ID1
through MSI Afterburner and starting mining, I can get an extra 1-2 MH/s from a card by simply lowering the memory frequency by 200 MHz once and then bringing it back to 1200/600

**Comment by AjlaaSS** [comment]
you can try to start miner while both cards are reseted to stock settings. after DAG file is verified and done, you can apply saved overclock in MSI A. see if that changes anything

but it shouldn't, I also have 3080 and it works fine when starting miner with already applied overclock.
it was only issue with older versions, prior to 39.6., in case of 3060 TI, to start with stock settings and than apply overclock to avoid invalid shares

**Comment by comet-bg** [comment]
If I start the miner before adjusting the OC settings, LHR is immediately activated.
In fact, my main question is why there is a hashrate difference if the two cards are in one miner and two cards work in separate miners?
